ddsrecorder_yaml/src/cpp/recorder/YamlReaderConfiguration.cpp

```diff
@@ -68,12 +68,16 @@ void RecorderConfiguration::load_ddsrecorder_configuration_(
         // Create Simple Participant Configuration
         simple_configuration = std::make_shared<SimpleParticipantConfiguration>();
         simple_configuration->id = "SimpleRecorderParticipant";
+        simple_configuration->app_id = "DDS RECORDER";
+        simple_configuration->app_metadata = "";
         simple_configuration->is_repeater = false;
 
         /////
         // Create Recorder Participant Configuration
         recorder_configuration = std::make_shared<ParticipantConfiguration>();
         recorder_configuration->id = "RecorderRecorderParticipant";
+        recorder_configuration->app_id = "DDS RECORDER";
+        recorder_configuration->app_metadata = "";
```

Comment:
Add a TODO for filling in metadata details once we've finalized the decision on the appropriate settings